src/windows7.rs

```diff
@@ -0,0 +1,37 @@
+//! Implementation for Windows 7 and 8
+//!
+//! For targets where we cannot use ProcessPrng (added in Windows 10), we use
+//! RtlGenRandom. See windows.rs for a more detailed discussion of the Windows
+//! RNG APIs (and why we don't use BCryptGenRandom). On versions prior to
+//! Windows 10, this implementation works, while on Windows 10 and later, this
+//! is a thin wrapper around ProcessPrng.
```

Comment:
So basically when the program is running on Windows 10 it is the same as what's in windows.rs for non-UWP targets, except it forces the loading of an additional DLL, and is less aesthetically pleasing? I think we should write down that expectation here so we remember it.

Reply:
Fixed to note that the windows7 implementation is identical modulo the loading of DLLs.